addons/website_event_sale/controllers/main.py

```diff
@@ -19,10 +19,6 @@ def _process_tickets_form(self, event, form_details):
         return res
 
     def _create_attendees_from_registration_post(self, event, registration_data):
-        # we have at least one registration linked to a ticket -> sale mode activate
-        if not any(info.get('event_ticket_id') for info in registration_data):
-            return super()._create_attendees_from_registration_post(event, registration_data)
-
```

Comment:
mhhh as far as I understand, when registering from the website, each registration will always be linked to a ticket because you always have this invisible input in the form :thinking: right ?
```
<input class="d-none" type="text" t-attf-name="#{counter}-event_ticket_id" t-attf-value="#{ticket['id']}"/>
```

Reply:
But maybe we can put this in a dedicated task/PR as it is not a priority ?

Reply:
The key itself is always set but it may be 0 from `'id': tid if ticket_dict.get(tid) else 0,`, which seems to indicate "this is a registration without a ticket". In that case `info.get('event_ticket_id')` would still be `Falsy`.

Unless I'm mixing flows but I don't think so. You can just create an event without ticket to test this. It will still let you register, it's just implicitly free :)

Admitedly it *is* a bit specific, perhaps we could always require events to have a ticket and make it free by default at some point :shrug: